.github/workflows/integration.yaml

```diff
@@ -39,7 +39,7 @@ jobs:
             runner: "Ubuntu_4C_16G"
     name: ${{ matrix.tox-environments }}
     runs-on: ${{ matrix.runner }}
-    timeout-minutes: 360
+    timeout-minutes: 60
```

Comment:
This might be too short depending on the test: `h-scaling, ha-integration and ha-networking-integration` often exceed that , I would prefer we keep as is for the time being.